dagger.json

```diff
@@ -2,6 +2,10 @@
   "name": "dagger-dev",
   "sdk": "go",
   "dependencies": [
+    {
+      "name": "gha",
```

Comment:
:thinking: seems like a mistake?

Reply:
@levlaz could you please check this comment?